.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,51 @@
+---
+name: Bug report
+about: Report an issue with TF2 Automatic
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+<!--
+Your issue may already be reported!
+Please search on the [issue tracker](https://github.com/Nicklason/tf2-automatic/search?type=Issues) before creating one.
+-->
+
+**Describe the bug**
+<!--
+A clear and concise description of what the bug is.
+-->
+
+**To Reproduce**
+Steps to reproduce the behavior:
+
+<!--
+For example
+1. Send trade offer '...'
+2. Add items '....'
+3. Click confirm '....'
+4. See error
+-->
+
+**Expected behavior**
+<!--
+A clear and concise description of what you expected to happen.
+-->
+
+**Screenshots / logs**
+<!--
+If applicable, add screenshots and the log file to help explain your problem.
+
+For errors, add a screenshot of the error. If that is not possible, then see the error log.
+
+Log files are located at `/tf2-automatic/logs`. There are three log files:
+* <steam account name>-<date>.log - Default log file
+* <steam account name>.error.log - Error log
+* <steam account name>.trade.log - Trade log
+-->
+
+**Additional context**
```

Comment:
I think this should just be additional notes

Reply:
No. The logs should always be included in a bug report.

Reply:
Or you mean that the title should be changed to "Additional notes"?

Reply:
Probably

Reply:
If that's the case then no, it is fine